README.md

```diff
@@ -265,6 +236,87 @@ The static site that gets built is pushed to the `documentation` branch of this
+correct date**
+2. **Take a look at the issues & PRs that are in the milestone**: does it look about right for the release highlights 
+we are planning to ship? Does it look like anything is missing? Don't worry about being aware of every PR that should 
+be in, but it's useful to take a moment to evaluate what's assigned to the milestone.
+3. **Post a message on the engineering Slack channel**, letting the team know you'll be the one cutting the upcoming 
+release, as well as:
+    1. Providing the link to the appropriate milestone
+    2. Reminding everyone to go over their issues and PRs and please assign them to the milestone
+    3. Reminding everyone of the scheduled date for the release
+
+#### Release day! 🚀
+
+1. **At the start of the day, post a small message on slack announcing release day!**. Communicate you'll be handling 
+the release, and the time you're aiming to start releasing (again, no later than 4pm, as issues may arise and 
+cause delays)
+2. **Go over the milestone and evaluate the status of any PR merging that's happening. Follow up with people on their 
+bugs and fixes.** If the release introduces new bugs or regressions that can't be fixed in time, we should discuss on 
+Slack about this and take a decision to go forward or postpone the release. The PR / issue owners are responsible for 
+communicating any issues which might be release relevant.
+3. Once everything in the milestone is taken care of, post a small message on Slack communicating you are about to 
+start the release process (in case anything is missing).
+4. **You may now do the release by following the instructions outlined in the 
+[Rasa Open Source README](#steps-to-release-a-new-version) !**
+
+### Steps to release a new version
+Releasing a new version is quite simple, as the packages are build and distributed by GitHub Actions.
+
+*Terminology*:
+* patch release (third version part increases): 1.1.2 -> 1.1.3
+* minor release (second version part increases): 1.1.3 -> 1.2.0
+* major release (first version part increases): 1.2.0 -> 2.0.0
+
+*Release steps*:
+1. Make sure all dependencies are up to date (**especially Rasa SDK**)
+    - For Rasa SDK that means first creating a [new Rasa SDK release](https://github.com/RasaHQ/rasa-sdk#steps-to-release-a-new-version) (make sure the version numbers between the new Rasa and Rasa SDK releases match)
+    - Once the tag with the new Rasa SDK release is pushed and the package appears on [pypi](https://pypi.org/project/rasa-sdk/), the dependency in the rasa repository can be resolved (see below).
+2. Switch to the branch you want to cut the release from (`master` in case of a major / minor, the current feature branch for patch releases) 
+    - Update the `rasa-sdk` entry in `pyproject.toml` with the new release version and run `poetry update`. This creates a new `poetry.lock` file with all dependencies resolved.
```

Comment:
sorry for creating another discussion 🙈 However, running `poetry update` seems to be enough, isn't it? We rarely  (never?) have any breaking changes in the SDK / Rasa Open Source interaction .

Reply:
I'm not sure I fully understand what you mean. Are you saying we don't need to do a Rasa SDK release for each Rasa Open Source release? 🤔

Reply:
> Are you saying we don't need to do a Rasa SDK release for each Rasa Open Source release?

Let's say we need the SDK releases because we want the version numbers to be the same. 

However, there is in my opinion no need to force the users to use the latest SDK version (which we would do by raising the lower bound in `pyproject.toml`) unless there is a breaking change. If there is no breaking change, then it'd be enough to run `poetry update` which would only update `poetry.lock` to use the latest SDK version. Does that make sense?

Reply:
Ohhh I see! But then we should have a way for people to know if there has been a breaking change. Will tests fail if I try to simply `poetry update` and there has been a breaking change, per example?

Reply:
As discussed yesterday, leaving this as is 😊